src/clientdynamicinfo.h

```diff
@@ -68,7 +68,7 @@ struct ClientDynamicInfo
 #ifndef SERVER
 	static v2f32 calculateMaxFSSize(v2u32 render_target_size, f32 gui_scaling) {
 		f32 factor =
-#ifdef HAVE_TOUCHSCREENGUI
+#ifdef __ANDROID__
```

Comment:
This should check for `enable_touch` directly because this code also does:

https://github.com/minetest/minetest/blob/5ac13ed7f0dd70a82631cbc9119792a3c59d08ed/src/gui/guiFormSpecMenu.cpp#L3285-L3292